dynamic-layers/phytec/recipes-bsp/u-boot/u-boot-imx_%.bbappend

```diff
@@ -14,16 +20,30 @@ SRC_URI += " \
     file://omnect_env_phycore_imx8mm.h \
 "
 
-# Appends a string to the name of the local version of the U-Boot image; e.g. "-1"; if you like to update the bootloader via
-# swupdate and iot-hub-device-update, the local version must be increased;
-#
-# Note: `UBOOT_LOCALVERSION` should be also increased on change of variables
-#   `APPEND`,
-#   `OMNECT_UBOOT_WRITEABLE_ENV_FLAGS`
-UBOOT_LOCALVERSION = "-6"
-PKGV = "${PV}${UBOOT_LOCALVERSION}"
+OMNECT_BOOTLOADER_CHECKSUM_FILES  = "${LAYERDIR_omnect}/classes/u-boot-scr.bbclass"
+OMNECT_BOOTLOADER_CHECKSUM_FILES += "${LAYERDIR_omnect}/recipes-bsp/u-boot/u-boot-scr.bb"
+OMNECT_BOOTLOADER_CHECKSUM_FILES += "${LAYERDIR_omnect}/recipes-bsp/u-boot/u-boot/*"
+OMNECT_BOOTLOADER_CHECKSUM_FILES += "${LAYERDIR_phytec}/recipes-bsp/u-boot/${PN}_${PV}_*.bb"
+OMNECT_BOOTLOADER_CHECKSUM_FILES += "${LAYERDIR_phytec}/recipes-bsp/u-boot/u-boot-*.inc"
+# included by "${LAYERDIR_phytec}/recipes-bsp/u-boot/${PN}_${PV}_*.bb" - how do we know it's still the case on update?:
+OMNECT_BOOTLOADER_CHECKSUM_FILES += "${LAYERDIR_core}/recipes-bsp/u-boot/u-boot.inc"
+OMNECT_BOOTLOADER_CHECKSUM_FILES += "${OMNECT_THISDIR_SAVED}/${PN}_%.bbappend"
+OMNECT_BOOTLOADER_CHECKSUM_FILES += "${OMNECT_THISDIR_SAVED}/u-boot/*"
+# imx-atf and u-boot are part of imx-boot(-phytec). (tbd: imx-boot(-phytec) recipe
+# is more or less copying thus currently not reflected here.)
+OMNECT_BOOTLOADER_CHECKSUM_FILES += "${LAYERDIR_phytec}/dynamic-layers/freescale-layer/recipes-bsp/imx-atf/imx-atf*.bbappend"
+OMNECT_BOOTLOADER_CHECKSUM_FILES += "${LAYERDIR_phytec}/dynamic-layers/freescale-layer/recipes-bsp/imx-atf/files/*"
+OMNECT_BOOTLOADER_CHECKSUM_FILES += "${LAYERDIR_fsl-bsp-release}/recipes-bsp/imx-atf/imx-atf_*.bb"
+
+OMNECT_BOOTLOADER_CHECKSUM_FILES_GLOB_IGNORE = "${LAYERDIR_omnect}/recipes-bsp/u-boot/.gitignore"
+OMNECT_BOOTLOADER_CHECKSUM_FILES_GLOB_IGNORE += "${LAYERDIR_omnect}/recipes-bsp/u-boot/u-boot/bootm_len_check.patch"
+OMNECT_BOOTLOADER_CHECKSUM_FILES_GLOB_IGNORE += "${LAYERDIR_omnect}/recipes-bsp/u-boot/u-boot/redundant-env-fragment.cfg"
+OMNECT_BOOTLOADER_CHECKSUM_FILES_GLOB_IGNORE += "${LAYERDIR_omnect}/recipes-bsp/u-boot/u-boot/redundant-env-fragment.cfg.template"
+# we don't use rauc and overwrite the u-boot env with omnect_env.patch
+OMNECT_BOOTLOADER_CHECKSUM_FILES_GLOB_IGNORE += "${LAYERDIR_phytec}/recipes-bsp/u-boot/u-boot-rauc.inc"
 
 inherit omnect_uboot_configure_env
+inherit omnect_bootloader_versioning
```

Comment:
is the order of the inherit part important? in recipes-bsp/u-boot/u-boot_%.bbappend it is different.... 
first
inherit omnect_bootloader_versioning
than
inherit omnect_uboot_configure_env

Reply:
no.
the order of calling `do_bootloader_checksum` before `do_configure` (and thus `omnect_uboot_configure_env`) is important and that's ensured in omnect_bootloader_version.bbclass.